src/utils/useWeb3Connect.js

```diff
@@ -24,14 +24,17 @@ import {
 } from './3BoxManager';
 
 const daiAbi = require('../abis/ERC20.json');
-const daoAbi = require('../abis/NoLossDao.json');
+const daoAbi = require('../abis/NoLossDao_v0.json');
+const depositAbi = require('../abis/PoolDeposits.json');
```

Comment:
All three of these files are symlinked to their originals in `../../contracts/deployed-builds/contracts`.

Actually, note to self, these are the only three files that need to be stored in the `deployed-builds` folder. The rest is just autogenerated junk from truffle that doesn't have use (and can be autogenerated when it does have a use in the future).

Reply:
Ok, removed all the other files, only those 3 are left now

Reply:
Let me know if the symlinks work for you, I actually (after thinking about it more) realise that symlinks probably use absolute paths rather than relative paths. I'll maybe completely remove the `deployed-builds` folder and keep the files in the `/abis/` directory only so that they are always up to date.

Reply:
Yeah, I guess you are right, just a simple script to copy them accross. I'll make a task for this